sources/packages/backend/libs/integrations/src/esdc-integration/disbursement-receipt-integration/disbursement-receipt.processing.service.ts

```diff
@@ -141,6 +150,51 @@ export class DisbursementReceiptProcessingService {
       }
     }
     result.processSummary.push(`Processing file ${remoteFilePath} completed.`);
+    const batchRunDate = responseData.header.batchRunDate;
+    result.processSummary.push(
+      `Processing provincial daily disbursement CSV file which are not sent on ${getISODateOnlyString(
```

Comment:
I had a small struggle to understand the sentence. I would recommend simplifying it as below.
`Processing provincial daily disbursement CSV file for ${getISODateOnlyString(batchRunDate)}.`